Podcast/SubscriptionsCollectionViewFlowLayout.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SubscriptionsCollectionViewFlowLayout.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscriptionsCollectionViewFlowLayout: UICollectionViewFlowLayout {
```

Comment:
you don't need this class, you can make the subscriptionsviewcontroller class implement the collectionviewflowlayoutdelegate protocol and implement all these as delegate functions.